cds_ils/demo_data/items.json

```diff
@@ -1,6 +1,10 @@
 [
   {
     "pid": "678e3-an678A",
+    "created_by": {
```

Comment:
This is also in the other PR of yours, might have conflict

Reply:
it was in invenio-app-ils